.gitlab-ci.yml

```diff
@@ -35,21 +35,23 @@ build_preview:
     URL: ${PREVIEW_DOMAIN}
     CONFIG: ${PREVIEW_CONFIG}
     MESSAGE: ":gift_heart: Your preview site is available!\nNow running tests..."
-    UNTRACKED_EXTRAS: "data,content/en/integrations"
   script:
     - post_dd_event "documentation deploy ${CI_COMMIT_REF_NAME} started" "${CI_PROJECT_URL}/pipelines/${CI_PIPELINE_ID}" "info"
     - version_static_assets
-    - sync_integration_descriptions
+    #- sync_integration_descriptions
+    - sync_integration_descriptions_cached
     - placehold_translations
     - build_hugo_site
     - build_hugo_site_ja
     - minify_html
-    - collect_static_assets
+    #- collect_static_assets
+    # remove service_checks json as we don't need to s3 push that..
+    - rm -rf data/service_checks
```

Comment:
should we remove the the `data/metrics.*` as well?

Reply:
So i only removed the service checks as they are `.json` files and the s3 upload command sends json files but we don't need to send these so it was actually slowing the upload by uploading files we didn't need to.